### PR TITLE
DOCS-116 Update the design of a new Homepage

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/assets/logo-dark-everest.svg
	new file:   docs/assets/logo-dark-pmm.svg
	modified:   docs/css/landing.css
	new file:   docs/percona-everest.md
	modified:   docs/percona-for-mongodb.md
	modified:   docs/percona-for-mysql.md
	modified:   docs/percona-for-postgresql.md
	modified:   docs/percona-lab.md
	modified:   docs/percona-operators.md
	modified:   docs/percona-utilities.md
	new file:   docs/pmm.md
	modified:   mkdocs.yml
	new file:   overrides/.icons/percona/everest.svg

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]

**Comment by nastena1606** [comment]
I believe we need a proper description for Everest and PMM on respective product pages

File: docs/percona-for-mysql.md [hunk 1]
```diff
@@ -12,17 +12,18 @@ hide:
 
 # Percona Software for MySQL
 
-Quickstart guides, user guides, and how-tos, release notes, and the latest downloads. Everything you need to optimize your MySQL databases is right here.
+Quickstart guides, user guides, and how-tos, release notes, and the latest downloads. Everything you need to optimize your MySQL databases is right here
```
Comment (patrickbirch) [inline docs/percona-for-mysql.md:15]: remove the first "and"
Quickstart guides, user guides, how-tos, release notes,

What does "and the latest downloads" mean?

**Review by patrickbirch — APPROVED** [review]

File: docs/percona-for-mysql.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/percona-for-mysql.md]: updated

**Review by nastena1606 — APPROVED** [review]
LGTM

**Pull request merged**